onnx_caffe2/frontend.py

```diff
@@ -271,14 +279,13 @@ def caffe2_init_net_to_initializers(init_net):
     return initializers
 
 
+def caffe2_net_to_onnx_model(net_def):
+    return make_model(caffe2_net_to_onnx_graph(net_def))
+
+
 def caffe2_net_to_onnx_graph(net_def):
-    if not net_def:
-        return None
     graph_def = onnx_pb2.GraphProto()
 
-    # to pass the check HasField('version')
-    graph_def.ir_version = onnx_pb2.IR_VERSION
```

Comment:
I don't see any setting of ir_version in model in this diff. We should set it!

Reply:
It's automatically set in `make_model` https://github.com/onnx/onnx/blob/master/onnx/helper.py#L43